chart/values.yaml

```
@@ -475,6 +475,14 @@ workers:
     # of local-path provisioner.
     fixPermissions: false
 
+  # If the celery stops heartbeating for 5 minutes (5*60s) kill the
+  # celery and let Kubernetes restart it
```

```suggestion
  # If the worker stops responding for 5 minutes (5*60s) kill the
  # worker and let Kubernetes restart it
```

nit: it's not really a heartbeat like the scheduler